python/semantic_kernel/core_plugins/sessions_python_tool/sessions_python_plugin.py

```diff
@@ -199,32 +212,32 @@ async def upload_file(
 
         remote_file_path = self._construct_remote_file_path(remote_file_path or os.path.basename(local_file_path))
 
-        with open(local_file_path, "rb") as data:
-            auth_token = await self._ensure_auth_token()
-            self.http_client.headers.update(
-                {
-                    "Authorization": f"Bearer {auth_token}",
-                    USER_AGENT: SESSIONS_USER_AGENT,
-                }
-            )
-            files = [("file", (remote_file_path, data, "application/octet-stream"))]
-
-            url = self._build_url_with_version(
-                base_url=self.pool_management_endpoint,
-                endpoint="python/uploadFile",
-                params={"identifier": self.settings.session_id},
-            )
-
-            response = await self.http_client.post(
-                url=url,
-                json={},
-                files=files,  # type: ignore
-            )
+        auth_token = await self._ensure_auth_token()
+        self.http_client.headers.update(
+            {
+                "Authorization": f"Bearer {auth_token}",
+                USER_AGENT: "SESSIONS_USER_AGENT",
```

Comment:
Is the value supposed to be the string `"SESSIONS_USER_AGENT"` or the variable `SESSIONS_USER_AGENT`?

Reply:
Great catch!! Not sure how that got "string-ified" :)